deepgram/transcription.py

```diff
@@ -56,6 +101,16 @@ def __init__(self, options: Options,
         self._queue: asyncio.Queue[Tuple[bool, Any]] = asyncio.Queue()
 
     async def __call__(self) -> 'LiveTranscription':
+        """
+        The __call__ function is a special method that allows the object to be called
+        as a function. In this case, it is used to connect the client and start the
+        transcription process. It returns itself after starting so that operations can
+        be chained.
+        
+        :param self: Used to Access the attributes of the class.
```

Comment:
Please remove this line from docstring `:param self: Used to Access the attributes of the class.`